packages/ag-charts-community/src/chart/interaction/zoomManager.ts

```diff
@@ -144,6 +148,18 @@ export class ZoomManager extends BaseManager<ZoomEvents['type'], ZoomEvents> imp
         this.applyChanges(callerId);
     }
 
+    panToBBox(callerId: string, seriesRect: BBox, target: BBoxValues) {
+        const { x: zoomX, y: zoomY } = this.getZoom() ?? {};
+        if (!zoomX || !zoomY) return;
```

Comment:
Could you clarify what you mean? At this point, `x` and `y` are of type `{ min: number, max: number } | undefined`.

My thoughts are that, when zooming is disabled, `x` and `y` would be `undefined` which lets us short-circuit and skip panning.

Reply:
Ah yeh, I was just thinking of the enabled case, so you're probably right it can skip the panning.

The only case might be if `x` is defined but y is not, like with just the navigator enabled?

Reply:
That's a good. I'll add explicit typing to ensure that `panToBBox` works when only 1 axis has zoom enabled